src/sorobandata_builder.js

```diff
@@ -154,6 +154,21 @@ export class SorobanDataBuilder {
     return this;
   }
 
+  /**
+   * Adds a set of ledger keys to the existing access footprint.
+   *
+   * @param {xdr.LedgerKey[]} readOnly  read-only keys added to access footprint
+   * @param {xdr.LedgerKey[]} readWrite read-write keys added to access footprint
+   *
+   * @returns {SorobanDataBuilder}
+   */
+  appendFootprint(readOnly, readWrite) {
```

Comment:
Alternative would be to add a method to the `xdr.LedgerFootprint.append(other: {readOnly?: []xdr.LedgerKey, readWrite?: xdr.LedgerKey[]}|xdr.LedgerFootprint)`, which would merge two footprints.
`concat`? `merge`?

Not sure which is better.

Reply:
I would really love to stick these into the object prototypes but it doesn't work? Or I'm doing something very wrong... Either way, `SorobanDataBuilder` already has this heh, so closing for now.